includes/Classifai/Admin/SavePostHandler.php

```diff
@@ -186,6 +187,26 @@ public function show_error_if() {
 		}
 	}
 
+	/**
+	 * Get the post statuses allowed to classify content
+	 *
+	 * @since 1.7.1 @TODO: Ensure this is updated to the version
+	 * @hook classifai_post_statuses
+	 *
+	 * @param string $post_type     The current post type.
+	 * @param int    $post_id       The current post ID.
+	 *
+	 * @return array $post_statuses The filtered array of post statuses.
+	 */
+	public function get_post_statuses( $post_type, $post_id ) {
+		// Defaults to published content only
+		$post_statuses = [ 'publish' ];
```

Comment:
We have a settings page where you can choose what post types you want to classify. Should we add post stati options there as well? This would allow users to easily choose those, with the default option there to be `publish`. Then the new filter here provides an override at a developer level if needed.

Reply:
I have added a settings section that allows post statuses to be selected.